test/ui-test/src/screens/SettingsScreen.py

```diff
@@ -52,7 +55,24 @@ def activate_open_wallet_section(self):
             verify_object_enabled(SidebarComponents.WALLET_ITEM.value)
            
         click_obj_by_name(MainScreenComponents.WALLET_BUTTON.value)
-     
+    
+    def delete_account(self, account_name: str):
+        click_obj_by_name(SidebarComponents.WALLET_ITEM.value)
+        
+        index = self._find_account_index(account_name)
+            
+        if index == -1:
+            raise Exception("Account not found")
+        
+        accounts = get_obj(WalletSettingsScreen.GENERATED_ACCOUNTS.value)
+        click_obj(accounts.itemAtIndex(index))
+        click_obj_by_name(WalletSettingsScreen.DELETE_ACCOUNT.value)
+        click_obj_by_name(WalletSettingsScreen.DELETE_ACCOUNT_CONFIRM.value)
+        
+    def verify_no_account(self, account_name: str):
+        index = self._find_account_index(account_name)
+        assert index == -1
```

Comment:
Maybe we can use squish validation api here by our wrapper, the squish driver's validation file `SquishDriverVerification`, so we can call one of its methods or add a new one that uses the api, so depending on the validation result, it direcly shows the step error in the squish console.

Reply:
Yeah, updated, it feel a it overkill to use squish has this has nothing to do with UI

Reply:
Why not? If you try to find an account that is not in the UI, it will fail so it is a test failure, right?